examples/jsm/postprocessing/AfterimagePass.js

```diff
@@ -89,6 +90,28 @@ class AfterimagePass extends Pass {
 
 	}
 
+	/**
+	 * Get the damping intensity. A higher value means a stronger after image effect.
+	 * 
+	 * @type {number}
+	 */
+	get damp() {
+
+		return this.uniforms[ 'damp' ].value;
+
+	}
+
+	/**
+	 * Set the damping intensity. A higher value means a stronger after image effect.
+	 * 
+	 * @type {number}
+	 */
+	set damp( value ) {
+
+		this.uniforms[ 'damp' ].value = MathUtils.clamp(value, 0.0, 1.0);
```

Comment:
`AfterImageNode` does not clamp the value and we do in general not sanitize input values. A note in the docs is sufficient.

Reply:
Clamp removed, added range to getter docs.